docs/source/clusters/kaizen/Kaizen-Ceph.md

```diff
@@ -0,0 +1,357 @@
+.rgw                541.0k                       3.0        1060T 0.0000                               1.0     32            on
+.rgw.buckets         6837G                       1.5        1060T 0.0094                               1.0     64            on
+openshift4          532.3M                       3.0        1060T 0.0000                               1.0     64            on
+.rgw.gc             173.1k                       3.0        1060T 0.0000                               1.0     16            on
+performance2         8271G                       3.0       74520G 0.3330                               1.0    256            on
+default.rgw.meta        0                        3.0        1060T 0.0000                               1.0     16            on
+.log                   28                        3.0        1060T 0.0000                               1.0     16            on
+m2                   6778G                       3.0        1060T 0.0187                               1.0    128            on
+default.rgw.log         0                        3.0        1060T 0.0000                               1.0     16            on
+.rgw.buckets.extra  132.4k                       3.0        1060T 0.0000                               1.0     32            on
+.rgw.buckets.index  46307k                       3.0        1060T 0.0000                               1.0     32            on
+.rgw.root            1090                        3.0        1060T 0.0000                               1.0     16            on
+rbd                 46038M                       3.0        1060T 0.0001                               1.0     32            on
+.users                121                        3.0        1060T 0.0000                               1.0     16            on
+.rgw.control            0                        3.0        1060T 0.0000                               1.0     16            on
+default.rgw.control     0                        3.0        1060T 0.0000                               1.0     16            on
+.usage              66180k                       3.0        1060T 0.0000                               1.0     32            on
+ostack2             42525G                       3.0        1060T 0.1175                               1.0   1024            on
+cephfsmeta          256.9M                       3.0        1060T 0.0000                               1.0     16            on
+cephfs              13303G                       3.0        1060T 0.0368                               1.0    128            on
+```
+
+## Rados Gateways
+
+We have 4 RGWs, setup in pairs of 2.
+
+The first pair, `kzn-rgw1` and `kzn-rgw2`, are used for OpenStack Swift and as such are configured to work with keystone. These have public IP addresses.
+
+The second pair, `kzn-rgw-j-01` and `kzn-rgw-j-02`, are used by the baremetal OpenShift 4.X cluster with the OCS operator. These hosts do not have a public IP since it's not required.
```

Comment:
These hostnames don't resolve on the mon hosts (or elsewhere). Should they at least be in the `/etc/hosts` on the mon hosts so that we can find them?